### ISSUE TITLE
seconds is printed twice.

### ISSUE THREAD

**Issue by shreydan** [ramonsaraiva/timy#10]
Tested with both Python 2.7 and 3.5 and on multiple devices.

```
>>>import timy
>>> 
>>> @timy.timer()
... def hello():
...     l = [x for x in range(0,1000)]
...     return l
... 
>>> hello()
Timy executed (hello) for 1 time in 0.000184 seconds seconds
Timy best time was 0.000184 seconds seconds
```

This doesn't happen with `with` statement

```
>>> import timy
>>> 
>>> with timy.Timer() as timer:
...     l = [x for x in range(0,10000)]
...     
... 
Timy 0.001751 seconds
```

**Comment by rasbt** [comment]
This should have been addressed in the PR yesterday (#8). I think that the PyPI version is still the old one so you'd have to install timy from the GitHub repo in the meantime until a new PyPI version gets pushed

**Comment by shreydan** [comment]
Oh. I think I should close this issue and install it from GitHub. Thank you!
